rustler_mix/mix.exs

```diff
@@ -2,21 +2,25 @@ defmodule Rustler.Mixfile do
   use Mix.Project
 
   def project do
-    [app: :rustler,
-     version: "0.21.0", # rustler_mix version
-     elixir: "~> 1.6",
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     name: "Rustler Mix",
-     source_url: "https://github.com/rustlerium/rustler",
-     homepage_url: "https://github.com/rusterlium/rustler",
-     deps: deps(),
-     docs: [
-       extras: ["guides/Basics.md"],
-       source_url_pattern: "https://github.com/rusterlium/rustler/blob/master/rustler_mix/%{path}#L%{line}"
-     ],
-     package: package(),
-     description: description()]
+    [
+      app: :rustler,
+      # rustler_mix version
```

Comment:
This might cause issues with the release scripts, eh?

Reply:
Were fixes necessary to resolve this?

Reply:
I learned that the `release.sh` script hasn't been working for a while. Probably after switching to workflows? We can revisit that in a separate PR.